File: Submissions/T07/H2.py
baris, kolom = map(int, input("Baris & Kolom : ").split())

matriks = []

for i in range(baris):
    row = list(map(int, input(f"Baris {i + 1} : ").split()))
    matriks.append(row)

print(f"Matriks ({baris}x{kolom}) :")
for i in range(baris):
    for j in range(kolom):
        print(matriks[i][j], end=" ")
    print()

print(f"Transpose ({kolom}x{baris}) :")
for i in range(kolom):
    for j in range(baris):
        print(matriks[j][i], end=" ")
    print()

File: Submissions/T16/H2.c
#include <stdio.h>

int main() {
    int n, m;
    printf("Baris & Kolom: ");
    scanf("%d %d", &n, &m);
    
    int matriks[n][m];
    
    for (int i = 0; i < n; i++) {
        printf("Baris %d: ", i + 1);
        for (int j = 0; j < m; j++) {
            scanf("%d", &matriks[i][j]);
        }
    }
    
    printf("Matriks (%dx%d):\n", n, m);
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    
    printf("Transpose (%dx%d):\n", m, n);
    for (int j = 0; j < m; j++) {
        for (int i = 0; i < n; i++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    
    return 0;
}